compiler/packages/babel-plugin-react-compiler/src/HIR/BuildHIR.ts

```diff
@@ -2427,6 +2427,15 @@ function lowerExpression(
       let expr = exprPath as NodePath<t.TSNonNullExpression>;
       return lowerExpression(builder, expr.get("expression"));
     }
+    case "MetaProperty": {
+      let expr = exprPath as NodePath<t.MetaProperty>;
+      return {
+        kind: "MetaProperty",
+        meta: expr.node.meta.name,
+        property: expr.node.property.name,
+        loc: expr.node.loc ?? GeneratedSource,
+      };
+    }
```

Comment:
i think what we can do is limit this to `import` meta properties for now, and throw a Todo error for other `meta` values.

Reply:
I constrained it to `expr.node.meta.name === "import" && expr.node.property.name === "meta"`